src/schema/search.ts

```diff
@@ -39,6 +44,7 @@ interface GQLSearchSuggestion {
   title: string;
   subtitle?: string;
   image?: string;
+  contentpreferencestatus?: ContentPreferenceStatus;
```

Comment:
`contentPreferenceStatus`?

Reply:
I had it like that, but due to graphorm lowercasing it need to be like this in return.
not 100% sure if we can fix it? 😅

Reply:
graphorm lowercasing? first time I hear about it, how does the other stuff work then like `contentPreference`